lib/src/port.dart

```diff
@@ -226,8 +226,10 @@ class _SerialPortImpl implements SerialPort {
   }
 
   static List<String> get availablePorts {
+    int rv;
     final out = ffi.allocate<ffi.Pointer<ffi.Pointer<sp_port>>>();
-    Util.call(() => dylib.sp_list_ports(out));
+    Util.call(() => rv = dylib.sp_list_ports(out));
```

Comment:
In #8 it was mentioned that `sp_list_ports()` returned `SP_ERR_FAIL` (-2). Wouldn't `Util.call()` throw a `SerialPortError` for that?

Reply:
No, for me on my android phone it did not throw an error